File: formulario_dinamico_back/src/main/java/com/formulario/dinamico/formulario_dinamico_back/dtos/TblArchivoDTO.java
package com.formulario.dinamico.formulario_dinamico_back.dtos;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
public class TblArchivoDTO {
    
    private Long idArchivo;

    private String nombreArchivo;

    private String archivo;
}

File: formulario_dinamico/lib/models/tblArchivoDTO.dart
class TblArchivoDTO {
  final int? idArchivo;
  final String? nombreArchivo;
  final String? archivo;

  TblArchivoDTO({ this.idArchivo, this.nombreArchivo, this.archivo});

  factory TblArchivoDTO.fromJson(Map<String, dynamic> json) {
    return TblArchivoDTO(
      idArchivo: json['idArchivo'],
      nombreArchivo: json['nombreArchivo'],
      archivo: json['archivo'],
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'idArchivo': idArchivo,
      'nombreArchivo': nombreArchivo,
      'archivo': archivo,
    };
  }

  @override
  String toString() =>
      'TblArchivoDTO(idArchivo: $idArchivo, nombreArchivo: $nombreArchivo, archivo: $archivo)';
}
